components/server/src/workspace/workspace-starter.ts

```diff
@@ -322,8 +326,10 @@ export class WorkspaceStarter {
             span.log({ newInstance: instance.id });
             instanceId = instance.id;
 
-            const result = await this.buildImageAndStartWorkspace({ span }, user, workspace, instance, envVars);
-            return result;
+            // start the instance
+            this.reconcileWorkspaceStart({ span }, instance.id);
```

Comment:
@geropl I remember you brought yesterday that it may break client expectations. Assume after testing it did not hold true with the dashboard?

Reply:
There one path in the dashboard ([reconnect](https://github.com/gitpod-io/gitpod/blob/68577dbc81bb8d5432287844b1989abc16047302/components/dashboard/src/start/StartWorkspace.tsx#L133)) that I think might not be covered. But my testing could not trigger it, yet.
But would love to test/debug again before we merge this.

Reply:
Turns out the current code handles this alright!